packages/plugin-aws-elasticsearch/__tests__/aws-elasticsearch.spec.ts

```diff
@@ -0,0 +1,24 @@
+const assert = require("assert");
+const { NotFoundError } = require("common-errors");
+import { Microfleet } from "@microfleet/core";
+
+describe("AWS Elasticsearch suite", () => {
+  let service: any;
+
+  it("should throw an error when plugin isn't included", async () => {
+    service = new Microfleet({
+      name: "tester",
+      plugins: [],
+    });
+    assert(!service.awsElasticsearch);
+  });
+
+  it("when log type is set to `service` and service does not include `logger` plugin, it throws an error", async () => {
+    assert.throws(() => {
+      service = new Microfleet({
+        name: "tester",
+        plugins: ["validator", "awsElasticsearch"],
+      });
+    }, NotFoundError);
+  });
```

Comment:
You may at least instantiate the service with a fully configured plugin, but not call .connect(), at least you will ensure that your plugin schema definition is fine or not.